client/src/reducers/datasets.js

```diff
@@ -19,6 +19,37 @@ function saveDatasetSettings(state, dataset) {
   });
 }
 
+function importDatasetPending(state, { importId, name }) {
+  const now = Date.now();
```

Comment:
Yes. I'll change this


On Mon, May 16, 2016 at 11:44 AM Gabriel Martin <notifications@github.com>
wrote:

> In client/src/reducers/datasets.js
> <https://github.com/akvo/akvo-lumen/pull/152#discussion_r63324530>:
>
> > @@ -19,6 +19,37 @@ function saveDatasetSettings(state, dataset) {
> >    });
> >  }
> >
> > +function importDatasetPending(state, { importId, name }) {
> > +  const now = Date.now();
>
> Perhaps we should grab the date in the action creator instead, to keep the
> reducers free of calls to non-pure functions? Same goes for line 36
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/akvo/akvo-lumen/pull/152/files/7bc7855df830996bf81ca82990683bfde93ea52d#r63324530>
>